client/core/core_test.go

```diff
@@ -6455,3 +6455,74 @@ func TestPreOrder(t *testing.T) {
 		t.Fatalf("PreOrder error after fixing everything: %v", err)
 	}
 }
+
+func TestRefreshServerConfig(t *testing.T) {
```

Comment:
go test -run="TestRefreshServerConfig" -coverprofile=cov.out && go tool cover -func=cov.out  | grep -E "refreshServerConfig"
PASS
coverage: 1.5% of statements
ok      decred.org/dcrdex/client/core   0.081s
decred.org/dcrdex/client/core/bookie.go:599:            refreshServerConfig                     100.0%